packages/types/lib/persist/api.ts

```diff
@@ -1,5 +1,23 @@
+import type { ApiError, Endpoint } from '../api.js';
+import type { MessageRowInsert } from '../logs/messages.js';
 import type { MergingStrategy, NangoRecord } from '../record/api.js';
 
+interface LogBody {
+    activityLogId: string;
+    log: MessageRowInsert;
+}
+
+export type PostLog = Endpoint<{
```

Comment:
Maybe we should prefix specific API to avoid conflict later, e.g: PostPersistLog or PersistPostLog, or else. wdyt?

Reply:
It is namespaced via the directory. I think it is ok to have the caller rename the import to avoid conflict if needed.